<launch>

  <arg name="uav_name" default="$(optenv UAV_NAME uav)"/>

  <group ns="$(arg uav_name)">
    
    <node pkg="uav_detect" name="uav_display" type="dbg_display_node" output="screen" >

      <rosparam file="$(find uav_detect)/config/sim_detection_params_F550.yaml" />
      <remap from="~depthmap" to="/$(arg uav_name)/realsense/camera/depth/image_raw" />
      <remap from="~rgb_img" to="/$(arg uav_name)/realsense/camera/color/image_raw" />
      <remap from="~blob_detections" to="/$(arg uav_name)/uav_detection/blob_detections" />
      <remap from="~processed_depthmap" to="/$(arg uav_name)/uav_detection/processed_depthmap" />

    </node>

  </group>

</launch>
